internal/captain/command.go

```diff
@@ -697,6 +697,13 @@ func setupSensibleErrors(err error) error {
 		return locale.NewInputError("command_flag_invalid_value", "", flagText, msg)
 	}
 
+	// pflag error of the form "flag needs an argument: <flag>, called at: "
+	if strings.Contains(errMsg, "flag needs an argument: ") {
+		flag := strings.SplitN(errMsg, ": ", 2)[1]
+		return locale.NewInputError(
+			locale.Tl("command_flag_needs_argument", "Flag needs an argument: [NOTICE]{{.V0}}[/RESET]", flag))
```

Comment:
I'm fairly sure that the call to `Tl` is redundant.

Reply:
Nevermind, that looks like an API issue. I'll look into it and file a story if needed.